adventure/api.py

```diff
@@ -63,5 +63,6 @@ def move(request):
 @csrf_exempt
 @api_view(["POST"])
 def say(request):
-    # IMPLEMENT
+    player = request.user.player
+    pusher.trigger(f'p-channel-{[player]}', u'broadcast', {'message':'Hello, world'})
```

Comment:
Does this broadcast to every player in the room? I think you should have a for loop in which you loop over each player defined in the currentPlayerUUIDs list variable (refer to line 51 of this file) you take that UUID and insert it where you have player in square brackets on line 67 and that should work properly if it does not work now.